metadata-io/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,19 @@
+<configuration debug="false" scan="true">
+
+    <appender name="STDOUT" target="System.out" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%date{ISO8601} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+            <level>WARN</level>
+        </filter>
+    </appender>
+
+    <logger name="com.datastax.oss" level="ERROR" />
```

Comment:
Fixing the logging for testcontainers results in noise from cassandra, pushing that down here.